src/target/riscv/riscv-013.c

```diff
@@ -3308,14 +3333,7 @@ static int maybe_execute_fence_i(struct target *target)
 	RISCV013_INFO(info);
 	RISCV_INFO(r);
 	if (info->progbufsize + r->impebreak >= 2) {
```

Comment:
This needs to >= 3 now that execute_fence() executes both a fence and a fence.i.